src/redux/form/sagas/formEdit.saga.ts

```diff
@@ -27,9 +27,22 @@ export function* editFilmTask(data: {
     const film = data.payload.film;
     const genre = data.payload.genre;
     const searchTitle = data.payload.searchTitle;
+    const offset = 0;
+    const sortingType = data.payload.sortingType;
```

Comment:
the same regarding destructuring and getting values from store here with select, without sending it from component